hgvs/normalizer.py

```diff
@@ -215,15 +215,19 @@ def _get_boundary(self, var):
                 left = exon_starts[i]
                 right = exon_ends[i]
 
-                if var.posedit.pos.end.base - 1 < cds_start:
+                if cds_start is None:
```

Comment:
Python 3 doesn't consider `None` comparable with `int`, while Python 2 considers `None < i` for all ints `i`. This preserves the Python 2 behavior in this method, though I didn't evaluate the old behavior for correctness.